concepts/null-undefined/about.md

```diff
@@ -0,0 +1,152 @@
+# About
+
+In contrast to many other languages, there are two different entities in JavaScript that represent the absence of a (meaningful) value.
+There is `null` and `undefined`.
+
+## Null
+
+The primitive value `null` is used as intentional "zero value" for a variable.
```

Comment:
"zero value" might be a bit misunderstood with the number zero. Can we change this to `"no value"` please?

```suggestion
The primitive value `null` is used as intentional "no value" for a variable.
```

Reply:
I thought this is a common term in programming but when I google it it seems rather specific to some languages. I applied your suggestion to all the files that have this part.